docs/08-resources/01-environments/03-testnet/index.md

```diff
@@ -144,9 +145,9 @@ The [faucet UI](./01-obtain-testnet-tokens.md "mention") provides users with a v
 | Contract                          | Address                                    |
 | --------------------------------- | ------------------------------------------ |
 | Bridge                            | [0x09379198A26E98b42E2619D2fCa1f3170c6FBB6e](https://www.oklink.com/amoy/0x09379198A26E98b42E2619D2fCa1f3170c6FBB6e) |
-| Fee Router                        | [0x055CEe6D441f0c913D863776356C12FfA2FCDc47](https://www.oklink.com/amoy/0x055CEe6D441f0c913D863776356C12FfA2FCDc47) |
-| Fixed Fee Handler                 | [0x43E3A09C5Db24130b94f415fD11bac4b1d44A6A3](https://www.oklink.com/amoy/0x43E3A09C5Db24130b94f415fD11bac4b1d44A6A3) |
-| Percentage Fee Handler            | [0xcdc03289407b570e434D4BA17B3481CF6B02f369](https://www.oklink.com/amoy/0xcdc03289407b570e434D4BA17B3481CF6B02f369) |
+| Fee Router                        | [0x04C4B75084B557ab0E79dE3DEd42b6c1dD4F48E6](https://www.oklink.com/amoy/0x04C4B75084B557ab0E79dE3DEd42b6c1dD4F48E6) |
+| Fixed Fee Handler                 | [0x5842071a7D59e56c73874194A1Ea25FcFB6e00D7](https://www.oklink.com/amoy/0x5842071a7D59e56c73874194A1Ea25FcFB6e00D7) |
+| Percentage Fee Handler            | [0x2E3d3b2535785341584D17840F59dc0370b3504D](https://www.oklink.com/amoy/0x2E3d3b2535785341584D17840F59dc0370b3504D) |
 | ERC-20 Handler                    | [0x7E462Fc0E0b6f5EC18C8935f2676E7E40e06a7af](https://www.oklink.com/amoy/0x7E462Fc0E0b6f5EC18C8935f2676E7E40e06a7af) |
 | Permissionless Generic Handler    | [0x3080Ecd6dEc3fe8a3A07d40e656f7D5fc6b3c3A7](https://www.oklink.com/amoy/0x3080Ecd6dEc3fe8a3A07d40e656f7D5fc6b3c3A7) |
 | Permissionless Generic Handler    | [0x1F932B2582c52f939DCfb367A3A42f8A95f66782](https://www.oklink.com/amoy/0x1F932B2582c52f939DCfb367A3A42f8A95f66782) |
```

Comment:
pls correct link to
`https://www.oklink.com/amoy/address/0x1F932B2582c52f939DCfb367A3A42f8A95f66782/contract`